docusaurus/docs/Android/04-guides/04-theming.md

```diff
@@ -1,4 +1,8 @@
-# Global Styling
+# Theming
+
+Many parts of the UI components can be changed and customized. It is possible to change colors, fonts, sizes and many other aspects of the `Views` to match a desired look. 
+
+## Global Styling
 
 If you want to customize view styles programmatically you can override the corresponding `StyleTransformer` in the `TransformStyle` class.
```

Comment:
```suggestion
Styles can be configured programmatically by overriding the corresponding `StyleTransformer` from the `TransformStyle` object.
```

Reply:
Also, it would be nice to explain that it overrides configuration from attributes